packages/admin/src/brevoConfiguration/BrevoConfigForm.tsx

```diff
@@ -149,6 +155,12 @@ export function BrevoConfigForm({ scope }: FormProps): React.ReactElement {
         return <Loading behavior="fillPageHeight" />;
     }
 
+    const validateFolderId = (value: number) => {
+        if (typeof value !== "number") {
+            return <FormattedMessage id="cometBrevoModule.brevoConfig.folderId.validation" defaultMessage="Please enter a number" />;
+        }
+    };
```

Comment:
Is this validation needed? Since you use a NumberField, you cannot enter other characters anyways right?